### DISCUSSION TITLE
140 open pull request

### DISCUSSION THREAD

**Discussion by BDVGitHub** [simple-icons/simple-icons#12458] [General]
How does it come that there are about 140 open pull requests? Is there something wrong with those pull requests?

**Comment by LitoMore** [comment]
We usually review these PRs in our spare time. This is a free open-source project, and we maintain it with passion (no income).

Before merging, we must verify its quality, sources, legal usage, and other metrics. This might take some time.

**Reply by BDVGitHub** [reply]
Thx for the explenation and efford.

Some (opensource) projects have te option to [buy me a cofee](https://buymeacoffee.com/), maybe it can also be used here?

**Reply by LitoMore** [reply]
We have an [Open Collective](https://opencollective.com/simple-icons) account.

Some of our maintainers also have their own GitHub Sponsors entry:

- https://github.com/sponsors/LitoMore
- https://github.com/sponsors/mondeja
- https://github.com/sponsors/runxel
- https://github.com/sponsors/service-paradis
